staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -172,6 +179,67 @@ var (
 		currentInflightRequests,
 		requestTerminationsTotal,
 	}
+
+	// these are the known (e.g. whitelisted/known) content types which we will report for
+	// request metrics. Any other RFC compliant content types will be aggregated under 'unknown'
+	knownMetricContentTypes = utilsets.NewString(
+		"application/apply-patch+yaml",
+		"application/font-ttf",
+		"application/font-woff",
+		"application/font-woff2",
```

Comment:
Really? Are these content types that the API server actually accepts, or just ones that we happened to get a request for?

Maybe these are an artifact of Kubernetes serving the dashboard? Or are these just provided when a browser hits one of the discovery endpoints?
- "application/vnd.ms-fontobject",
- "application/x-font-woff",
- "font/ttf",
- "font/woff",
- "font/woff2",
- "image/gif",
- "image/jpeg",
- "image/png",
- "image/svg+xml",
- "image/vnd.microsoft.icon",
- "image/x-icon",

Also, these look especially suspect:
- "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
- "application/pdf",

I suppose from the perspective of bounding the content types, it doesn't really matter.

Reply:
They are possibly artifacts of proxied requests (or the dashboard as you point out). Either way, I'm okay with leaving them in the list or removing them. I don't have a strong preference on it.

Reply:
@tallclair do you have a preference?

Reply:
Let's leave them for now, to err on the side of caution.

Reply:
👍